RULES.md

```diff
@@ -2181,17 +2182,53 @@ A route's color and `route_text_color` should be contrasting.
 
 </details>
 
-<a name="RouteShortAndLongNameEqualNotice"/>
+<a name="RouteLongNameConainsShortName"/>
 
-### route_short_and_long_name_equal
+### route_long_name_contains_short_name
 
-A single route has the same values for `route_short_name` and `route_long_name`.
+Long name can not contain short name for a single route.
```

Comment:
Why can't the long name contain the short name for a single route? it might be worth explaining the reasoning behind this requirement.

Reply:
It's a common pattern for most transit applications to display the combination of route_short_name + route_long_name as a combined label for a route, so if the route_long_name contains the short name, it's duplicative.

Reply:
Should we add to the description to explain this?